modules/client/src/internal/utils.ts

```diff
@@ -237,6 +245,48 @@ export function toTokenTransfer(transfer: SubgraphTransferListItem): Transfer {
   }
 }
 
+export function toPluginRepoRelease(
+  release: SubgraphPluginRepoReleaseListItem,
+  metadata: PluginRepoReleaseMetadata,
+): PluginRepoRelease {
+  return {
+    release: release.release,
+    currentBuild: Math.max(...release.builds.map((build) => build.build)),
+    metadata,
+  };
+}
+
+export function toPluginRepoListItem(
+  pluginRepo: SubgraphPluginRepoListItem,
+  releases: PluginRepoRelease[],
+): PluginRepoListItem {
+  return {
+    address: pluginRepo.id,
+    subdomain: pluginRepo.subdomain,
+    releases,
+  };
+}
+export function toPluginRepo(
+  pluginRepo: SubgraphPluginRepo,
+  releaseMetadata: PluginRepoReleaseMetadata,
+  buildMetadata: PluginRepoBuildMetadata,
+): PluginRepo {
+  return {
+    address: pluginRepo.id,
+    subdomain: pluginRepo.subdomain,
+    current: {
+      build: {
+        metadata: buildMetadata,
+        number: pluginRepo.releases[0].builds[0].build,
```

Comment:
Are we doing any assumptions here regarding the content?

Reply:
No we are not the query specifies, the sorting and the limit of both releases and builds so only one release and only one build are returned

Reply:
If so, let's add a comment explaining that we assume that there will always be one release and build entry. 
BTW, are we 100% sure that the query will always give us the latest release/build?

Reply:
Yes it sorts by release / build in descending order and que only pick the first one, so it should be fine